biogrid/libs/components/src/lib/components.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Components from './components';
+
+describe(' Components', () => {
+  it('should render successfully', () => {
+    const { baseElement } = render(<Components />);
+    // expect(baseElement).toBeTruthy();
```

Comment:
why is this commented out? If it is removed, this isn't really testing anything right?

Reply:
Oh yes, I was having linter issues, I will put that back in

Reply:
Added back, it was because of a linter issue. Fixed now